src/scripts/operators/UserOperators.js

```diff
@@ -0,0 +1,10 @@
+/* @flow */
+
+import User from 'records/UserRecords'
+
+export const getEmail = (user: User): string => user.get('email')
+
+export const getEmailIsVerified = (user: User): boolean =>
+  user.get('emailIsVerified')
+
+export const getName = (user: User): string => user.get('name')
```

Comment:
I discussed this w/ Elia, but I like the idea of using `operators` in our codebase. Essentially they take a record as an input and return any output. Useful in selectors and components alike.